safeeyes/tests/test_model.py

```diff
@@ -0,0 +1,354 @@
+# Safe Eyes is a utility to remind you to take break frequently
+# to protect your eyes from eye strain.
+
+# Copyright (C) 2017  Gobinath
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import random
+from safeeyes import model
+
+class TestBreak:
+    def test_break_short(self):
+        b = model.Break(
+            model.BreakType.SHORT_BREAK,
+            "test break",
```

Comment:
It is advised to use named argument to improve readability. For example, `None` could be anything, but `image=None` leaves no questions.